WindowState/README.md

```diff
@@ -0,0 +1,127 @@
+# WindowState - Surface Duo Compose SDK
+
+**WindowState** is a component for Jetpack Compose, help you easily get details about the window state of the dual-screen, foldable and large screen devices, using the Google [Jetpack WindowManager](https://developer.android.com/jetpack/androidx/releases/window) library.
+
+The component provides the current window information as a Compose state, including folding position, orientation and window size classes. For the dual-screen and foldable device, combining different folding position and orientation, we introduce four display postures to take advantage of these new form factors, Dual Portrait, Dual Landscape, Single Portrait, Single Landscape. 
```

Comment:
Another small comment, I just have suggestions for how to reword this a little:
"...For dual-screen and foldable devices, combining different folding positions and orientations, we introduce four display postures to take advantage of these new form factors: Dual Portrait, Dual Landscape, Single Portrait, Single Landscape. "